DSharpPlus/Clients/DiscordClient.WebSocket.cs

```diff
@@ -137,7 +137,7 @@ internal async Task InternalConnectAsync()
             this._webSocketClient.ExceptionThrown += SocketOnException;
 
             var gwuri = new QueryUriBuilder(this.GatewayUri)
-                .AddParameter("v", "9")
+                .AddParameter("v", "10")
```

Comment:
I wonder why this doesn't use the same CONST var as in Endpoints.cs, as both are the same at all times (we don't mix and match API/GW version anyway)

Reply:
`Endpoints.API_VERSION`